docs/topics/downloader-middleware.rst

```diff
@@ -1070,9 +1070,14 @@ In order to use this parser:
 
 * Install `Reppy <https://github.com/seomoz/reppy/>`_ by running ``pip install reppy``
 
+.. warning:: `Upstream issue #122
+  <https://github.com/seomoz/reppy/issues/122>`_ prevents reppy usage in
+  Python 3.9+.
```

Comment:
The warning should be nested into the list item, I think:

```suggestion
* Install `Reppy <https://github.com/seomoz/reppy/>`_ by running ``pip install reppy``

  .. warning:: `Upstream issue #122
     <https://github.com/seomoz/reppy/issues/122>`_ prevents reppy usage in
     Python 3.9+.
```

Reply:
@Gallaecio Thanks for the review.Actually I have seen that in the whole documentation where were are defining the functionalities of the component the blocks were nested there(which will make warning block appear in the middle of the page horizontally).
for example:
-Functionality 1
    - Details of functionality 
    - some block(which will appear in the middle)

But in our scenario here we an unordered list.
* some reference
* some reference 2
* some reference 3

PS: I can make it into a nested structure if you want that or I can post the screenshot if you say and let you decide which one is better.

Reply:
link for without nesting of warning:
[https://ibb.co/hRq7jwL](url)

link for nesting of warning:
[https://ibb.co/khz8VX7](url)

Reply:
@Gallaecio  please review it and let me know. Thanks alot.

Reply:
I prefer it either nested to the installation step (because it’s a warning about an installation detail), or as I originally suggested: non-nested at the beginning of the whole reppy section, right under the header (since users may not want to even consider reppy knowing this issue).

Reply:
yes right. Great. Thanks.

Reply:
Done.